Prace_domowe/Praca_domowa1/Grupa1/Maciej_Chylak/PD1_Maciej_Chylak.ipynb

```diff
@@ -0,0 +1,2188 @@
+{
```

Comment:
<p>Nie mamy w danych szeregu czasowego, dlatego taka wizualizacja jest mało informatywna - istnieją lepsze metody wizualizacji (odnosi się to także do wykresu poniżej).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>